rs-react-app/src/components/handleForm/handleForm.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+interface handleFormProps {
+  onClick?: (number: number | string) => void
+}
+
+interface handleFormState {
+  inputValue: string, 
+  searchValue: boolean,
+}
+
+const HandleForm = (props: handleFormProps, ) => {
```

Comment:
good place to do destructurization
```suggestion
const HandleForm = ({ onClick }: handleFormProps, ) => {
```